docs/README-AWS.md

```diff
@@ -398,70 +483,97 @@ inputs:
           es_datastream_name: "logs-generic-default"
 ```
 
-#### Notes
-`inputs.[].expand_event_list_from_field` can be defined as string with the value of a key in the JSON that contains a list of elements that must be sent as events instead of the encompassing JSON.
+**Notes:**
+
+`inputs.[].expand_event_list_from_field` can be defined as a string with the value of a key in the JSON that contains a list of elements that must be sent as events instead of the encompassing JSON.
 
-Beware that when relying on the [Routing support for AWS Service Logs](#routing-support-for-aws-services-logs), any value set for `expand_event_list_from_field` configuration param will be ignored, since it will be taken care automatically by the Elastic Serverless Forwarder.
+You should note that when relying on the [Routing AWS Service Logs](#routing-aws-service-logs), any value set for the `expand_event_list_from_field` configuration parameter will be ignored, because this will be automatically handled by the Elastic Serverless Forwarder.
+
+### Example
+
+With the following input:
 
-#### Example:
-Having the following content from the input:
 ```json lines
 {"Records":[{"key": "value #1"},{"key": "value #2"}]}
 {"Records":[{"key": "value #3"},{"key": "value #4"}]}
 ```
-- without setting `expand_event_list_from_field` two events will be forwarded:
+
+Without setting `expand_event_list_from_field`, two events will be forwarded:
+
     ```json lines
     {"@timestamp": "2022-06-16T04:06:03.064Z", "message": "{\"Records\":[{\"key\": \"value #1\"},{\"key\": \"value #2\"}]}"}
     {"@timestamp": "2022-06-16T04:06:13.888Z", "message": "{\"Records\":[{\"key\": \"value #3\"},{\"key\": \"value #4\"}]}"}
     ```
 
-- if `expand_event_list_from_field` value is set to `Records`, four events will be forwaderd:
+If `expand_event_list_from_field` is set to `Records`, four events will be forwarded:
+
     ```json lines
     {"@timestamp": "2022-06-16T04:06:21.105Z", "message": "{\"key\": \"value #1\"}"}
     {"@timestamp": "2022-06-16T04:06:27.204Z", "message": "{\"key\": \"value #2\"}"}
     {"@timestamp": "2022-06-16T04:06:31.154Z", "message": "{\"key\": \"value #3\"}"}
     {"@timestamp": "2022-06-16T04:06:36.189Z", "message": "{\"key\": \"value #4\"}"}
     ```
```

Comment:
And I think these are indented too far (compare to the code block above). 😛 

<img width="654" alt="image" src="https://user-images.githubusercontent.com/10479155/176272406-cd97fa6e-5f6c-49b8-bda8-49db6ffd6fa3.png">